themes/bootstrap3/templates/_ui/components/menu-button.phtml

```diff
@@ -30,7 +30,12 @@
   <ul class="dropdown-menu">
     <?php foreach ($this->menuItems as $current): ?>
       <li class="dropdown__item <?php if ($current['selected'] ?? false): ?> active<?php endif ?>">
-        <a class="dropdown__link" href="<?=$this->escapeHtmlAttr($current['url']) ?>" rel="nofollow">
+        <a class="dropdown__link"
+          href="<?=$this->escapeHtmlAttr($current['url']) ?>"
+          <?php if ($current['external'] ?? false): ?> target="_blank"<?php endif; ?>
```

Comment:
Might it make sense to allow the target attribute to be set directly here instead of abstracting it with a boolean `external`? I admit that I can't really think of a use case for setting a different target, but thought I'd raise the question while we're working on it, just in case there's a reason to do it. Another option might be to allow the `external` option to accept a boolean or a string, and use the string directly but convert boolean true to `_blank`.

Reply:
Fair enough. I suppose for now we can merge this as-is, and if a use case arises, we can always introduce linkAttributes and come up with a way to reconcile that with other settings. At that point, it might make sense to refactor the code to use the `htmlAttributes()` helper, since array manipulation would make reconciliation simpler.

Reply:
I think that _not_ allowing an arbitrary target is a good thing. It's way too easy to mistype `_blank` and get a named window that causes all sorts of trouble.